### ISSUE TITLE
Play animation only once?

### ISSUE THREAD

**Issue by tncball33** [facebookincubator/Keyframes#86]
Is there a way to make the animation play only once? Currently everything defaults to repeating.

**Comment by skagedal** [comment]
Not sure which platform you are using, but on iOS you can set `repeatCount` to 1 on the `KFVectorLayer`. This is a property inherited from `CALayer`, [see here](https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/Animation_Types_Timing/Articles/Timing.html). You do however have to set this on the layer before setting the face model (it would be nice if this could be modified afterwards).

**Comment by m1ga** [comment]
And on Android you can use `kfDrawable.stopAnimationAtLoopEnd();` after you started the animation. But a general `playOnce()` method that handles both platforms would be nice to have

**Comment by tncball33** [comment]
Thanks guys, yeah I was trying to set the repeatCount to 1. Didn't realize you had to set it before setting the faceModel. Problem looks to be solved. I also agree there should probably be some sort of general playOnce() method exposed on both platforms.

**Comment by m1ga** [comment]
Android PR: https://github.com/facebookincubator/Keyframes/pull/94
